content/en/integrations/guide/oracle-check-upgrade-7.53.0.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Oracle Integration in the Agent 7.53.0
+kind: guide
+---
+
+# What has changed?
+
+- The Oracle integration is completely rewritten in the Agent release `7.53.0`.
+
+- The new check is easier to set up, and is less prone to configuration errors because it doesn't require any external clients such as Oracle JDBC drivers. The `jdbc` related parameters are ignored in the new check.
+
+- The new Oracle check emits the same metrics as the deprecated Oracle integration.
+
+- System metrics in multi-tenant databases are emitted for each pluggable database.
+
+- The metrics are emitted with the database host name as recorded in the system view `v$instance.host_name` instead of the Agent host name. That can break dashboards and monitors which are based on the host name tag.
+
+# Required action
+
+The configuration in the subdirectory `oracle.d` remains the same.
+
+The Oracle new check requires more read privileges on system views in the database than the deprecated Oracle integration. Run the `grant` commands for you hosting type prior to upgrading the agent.
```

Comment:
Need capital "A" in "Agent".

Reply:
Also to make this flow more seamlessly into the next sections, could we include a colon instead of a period at the end of this sentence?